src/main/java/core/basesyntax/Application.java

```diff
@@ -1,10 +1,10 @@
 package core.basesyntax;
 
 public class Application {
-    public static void main(String[] args) {
-        FigureSupplier figureSupplier = new FigureSupplier();
-        Figure[] figures = new Figure[6];
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
+    private static final Figure[] figures = new Figure[6];
```

Comment:
6 is a magic number, so please make it a CONSTANT.

Reply:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/140905715/aa129f0e-0277-4c70-80b2-b4dace48c278)